src/Solver/DualDecomp/DdWorkerUB.cpp

```diff
@@ -364,8 +432,17 @@ DSP_RTN_CODE DdWorkerUB::solve() {
 				CoinMin(CoinMax(0.01, time_remains_),
 				par_->getDblParam("DD/SUB/TIME_LIM")));
 
-		/** solve */
-		osi_[s]->solve();
+		// /** solve */
+		// osi_[s]->solve();
+		if (model_->isQuadratic())
+		{
+			/* solve using MIQCP solver */
+			osi_[s]->solveQp();
```

Comment:
Thought: Can this be handled in `DspOsi` while using `solve()` only? Maybe `DspOsi` can have a flag to indicate whether the problem is quadratic or not..?

Reply:
Merged solveQp() into solve() in DspOsi